specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json

```diff
@@ -907,5 +916,16 @@
         }
       }
     }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Connection string or HTTP or HTTPS endpoint for the WebPubSub service instance.",
```

Comment:
Sounds a little confusing to me that this Endpoint can be also a connection string

Reply:
Endpoint sounds enough? Connection string needs parser or convenience layer after all

Reply:
```suggestion
      "description": "HTTP or HTTPS endpoint for the WebPubSub service instance.",
```

Reply:
good point @vicancy , updating!